README.md

```diff
@@ -75,7 +81,7 @@ Docker-compose (https://docs.docker.com/compose/install/)
 
 ### How to run
 Prepare environment and start by running
-`STAGE=prod ./prepare_and_start.sh` for production 
+`STAGE=prod ./prepare_and_start.sh` for production
 `STAGE=testing ./prepare_and_start.sh` for testing.
```

Comment:
@edisinovcic Possibly this `How to run` section should be about production only? Mostly because prepare_and_start copy files around in the repo and these end up in the git diff. 

In this PR, I modified docker-compose-testing.yaml to use env.testing directly and using the original `up` command as described below seemed to launch everything correctly for manual testing...

Reply:
The only problem with it is that docker-compose uses .env file to enter values in docker-compose and that .env.testing environment is send to dockerfiles during build. There is no way to say docker-compose that it needs to use .env.something while setting up parameters in docker-compose.yaml file. We can maybe add to .gitignore docker-compose.yaml and .env so they can be created. 

There is no difference as ports are the same on .env.prod and .env.testing currently. 

Diff is definitely a problem and those changes are not needed there. I'm open to proposals here if you think this can be done better. :)

Reply:
Would something like this be better? https://github.com/ethereum/source-verify/pull/55

Reply:
> There is no way to say docker-compose that it needs to use .env.something while setting up parameters in docker-compose.yaml file. 

Ah I see..